geometry_msgs/Pose[] poses
geometry_msgs/Point object_position

float64[] times